zeroex/order_test.go

```diff
@@ -58,7 +59,22 @@ func TestMarshalUnmarshalOrderEvent(t *testing.T) {
 		SignedOrder:              signedOrder,
 		EndState:                 ESOrderAdded,
 		FillableTakerAssetAmount: big.NewInt(2000),
-		ContractEvents:           []*ContractEvent{},
+		ContractEvents: []*ContractEvent{
```

Comment:
@fabioberger we didn't catch this bug because we never tried marshaling/unmarshaling an `OrderEvent` with a non-empty `ContractEvents` field.